extensions/amp-subscriptions/0.1/renderer.js

```diff
@@ -52,7 +57,7 @@ export class Renderer {
    * @private
    */
   setState_(type, state) {
-    this.vsync_.mutate(() => {
+    this.resources_.mutateElement(this.getRootElement_() , () => {
```

Comment:
Please use `this.ampdoc_.getBody()` instead here.

Reply:
Done